packages/react-core/src/components/ContextSelector/examples/ContextSelector.md

```diff
@@ -85,6 +86,70 @@ class SimpleContextSelector extends React.Component {
 }
 ```
 
+### Plain with text
+
+```ts
+import React from 'react';
+import { ContextSelector, ContextSelectorItem } from '@patternfly/react-core';
+
+const items = [
+  'My Project',
+  'OpenShift Cluster',
+  'Production Ansible',
+  'AWS',
+  'Azure',
+  'My Project 2',
+  'OpenShift Cluster ',
+  'Production Ansible 2 ',
+  'AWS 2',
+  'Azure 2'
+];
+
+const PlainTextContextSelector: React.FC = () => {
+  const [isOpen, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState(items[0]);
+  const [searchValue, setSearchValue] = React.useState('');
+  const [filteredItems, setFilteredItems] = React.useState(items);
+
+  function onToggle(event: any, isOpen: boolean) {
+    setOpen(isOpen);
+  };
+
+  function onSelect(event: any, value: string) {
+    setSelected(value);
+    setOpen(!isOpen);
+  };
+
+  function onSearchInputChange(value: string) {
+    setSearchValue(value);
+  };
+
+  function onSearchButtonClick(event: any) {
```

Comment:
The event type here should match the API. The type is `event: React.SyntheticEvent<HTMLButtonElement>`